src/python/WMCore/WebTools/RESTFormatter.py

```diff
@@ -26,7 +28,21 @@ def __init__(self, config):
 
         TemplatedPage.__init__(self, config)
 
+    def genstreamer(self, data):
+        yield "["
+        firstItem  = True
+        for item in data:
+            if not firstItem:
+                yield ","
```

Comment:
Ah, wait, you are also guaranteed to yield the actual value, right? So no problem